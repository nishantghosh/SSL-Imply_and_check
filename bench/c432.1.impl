# Some test implications

fault_site(191, 1)
imply(191, 0)
J_frontier
imply(254, 1)
imply(357, 0)
J_frontier
D_frontier
x-path
display_lines
