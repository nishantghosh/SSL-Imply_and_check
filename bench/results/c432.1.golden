Parsing ISCAS Format... done.
Parsing commands file... done.
J-Frontier
191
$

J-Frontier
357
254
191
$

D-Frontier
276
$

X-PATH
276
$

Circuit State
1: X
102: X
105: X
108: X
11: X
112: X
115: X
118: X
119: X
122: X
123: X
126: X
127: X
130: X
131: X
134: X
135: X
138: X
139: X
14: X
142: X
143: X
146: X
147: X
150: X
151: X
154: X
157: X
158: X
159: X
162: X
165: X
168: X
17: X
171: X
174: X
177: X
180: X
183: X
184: X
185: X
186: X
187: X
188: X
189: X
190: X
191: DBAR
192: X
193: X
194: X
195: X
196: X
197: X
198: X
199: X
203: X
21: X
213: X
223: X
224: X
227: X
230: X
233: X
236: X
239: X
24: X
242: X
243: X
246: X
247: X
250: X
251: X
254: 1
255: X
256: X
257: X
258: X
259: X
260: X
263: X
264: X
267: X
27: X
270: X
273: X
276: X
279: X
282: X
285: X
288: X
289: X
290: X
291: X
292: X
293: X
294: X
295: X
296: X
30: X
300: X
301: X
302: X
303: X
304: X
305: X
306: X
307: X
308: X
309: X
319: X
329: X
330: X
331: X
332: X
333: X
334: X
335: X
336: X
337: X
338: X
339: X
34: X
340: X
341: X
342: X
343: X
344: X
345: X
346: X
347: X
348: X
349: X
350: X
351: X
352: X
353: X
354: X
355: X
356: X
357: 0
360: 1
37: X
370: 1
371: X
372: X
373: X
374: X
375: X
376: X
377: X
378: X
379: X
380: X
381: X
386: X
393: X
399: X
4: X
40: X
404: X
407: X
411: X
414: X
415: X
416: X
417: X
418: X
419: X
420: X
421: X
422: X
425: X
428: X
429: X
43: X
430: X
431: X
432: X
47: X
50: X
53: X
56: X
60: X
63: X
66: X
69: X
73: X
76: X
79: X
8: X
82: X
86: X
89: X
92: X
95: X
99: X
$

